packages/react-composites/package.json

```diff
@@ -20,7 +20,7 @@
     "build:e2e:call": "rushx _by-flavor \"webpack-cli build --config ./tests/app/call/webpack.config.js --mode=production --env production\"",
     "build:e2e:chat": "rushx _by-flavor \"webpack-cli build --config ./tests/app/chat/webpack.config.js --mode=production --env production\"",
     "build:e2e:callwithchat": "rushx _by-flavor \"webpack-cli build --config ./tests/app/callwithchat/webpack.config.js --mode=production --env production\"",
-    "test": "rushx _by-flavor \"rushx _test:by-flavor\"",
+    "test": "rush build -t . && rushx _by-flavor \"rushx _test:by-flavor\"",
```

Comment:
I got the purpose of adding rush build -t here.
Somehow call rush build sounds like a parent level command

Imagine if you are doing a `rush test`, this `rush build -t` will run for every child repo which might cause some duplication, I guess we just need to give developers expectation that we don't do a full build for the test here

Reply:
Hmm.. Actually, I read this wrong earlier. `rushx build` would have been an acceptable trade-off. Running `rush build -t` when developer ran `rushx test` is wrong -- the expectation for a `rushx` command is that it is packlet-local only.

Reply:
Let's remove this from this PR and figure out if we need better documentation / understand why the test failed without the `rush build -t`.